docs/src/sphinx/deploy.rst

```diff
@@ -57,12 +57,21 @@ repository-uri
     In order to use key-based authentication, the private key of the system on which
     dbuild runs must be in ``~/.ssh/id_rsa``; the passphrase must be empty.
 
+  ``bintray:/owner/repo/package/version[#release]``
```

Comment:
@pvlugter FYI  - This will make publishing to bintray easier in RP, but we haven't merged it yet.   THis related to the email I sent you about having to manually click the publish button for your repository.